client/dive-common/components/RunTrainingMenu.vue

```diff
@@ -118,47 +118,65 @@ export default defineComponent({
     </template>
 
     <template>
-      <v-card v-if="trainingConfigurations">
+      <v-card
+        v-if="trainingConfigurations"
+        outlined
+      >
         <v-card-title class="pb-1">
           Run Training
         </v-card-title>
 
         <v-card-text>
-          Specify the name of the resulting pipeline
-          and configuration file to use for training.
+          <p>
+            Specify the name of the resulting pipeline
+            and configuration file to use for training.
+          </p>
+          <v-alert
+            dense
+            color="warning"
+            outlined
+          >
+            This instance is updated on Thursday at 2AM EST.
+            If your training job is running at that time it may be restarted or killed.
+          </v-alert>
+
+          <v-text-field
+            v-model="trainingOutputName"
+            outlined
+            hide-details
+            class="my-4"
+            label="Output Name"
+          />
+          <v-select
+            v-model="selectedTrainingConfig"
+            outlined
+            hide-details
+            class="my-4"
+            label="Configuration File"
+            :items="trainingConfigurations.configs"
+          />
+          <p>
+            Input data:
+          </p>
+          <v-chip
+            v-for="id in selectedDatasetIds"
+            :key="id"
+            small
+            class="mr-1 mb-2"
+          >
+            <pre>{{ id }}</pre>
```

Comment:
I like it, but these numbers only mean stuff you and I and people who understand the routing.  If it was the folder name that might help a bit more for regular users.

Reply:
It wasn't available without adding more props.  I agree it's a bit odd.  Mostly I wanted a clear visual indicator of quantity.

I could just change it to say "Train on {count} datasets" if that's preferred.